scripts/tx-filter-stream.js

```diff
@@ -95,6 +99,12 @@ async function main() {
     emitBlockEventToFilterCollection,
   );
 
+  // Send transaction instant locks via `subscribeToTransactionsWithProofs` stream
+  dashCoreZmqClient.on(
+    dashCoreZmqClient.topics.rawtxlock,
```

Comment:
You need to work with `rawtxlocksig` because `rawtxlock` doesn't have the signature
https://github.com/dashpay/dash/blob/master/src/zmq/zmqnotificationinterface.cpp#L50
https://github.com/dashpay/dash/blob/master/src/zmq/zmqpublishnotifier.cpp#L220-L231

Reply:
I see! Thanks you!